scripts/gen_psql_tests

```diff
@@ -195,6 +195,7 @@ commands = set([
     "UPDATE",
     "VACUUM",
     "VALUES",
+    "WITH"
```

Comment:
This is fine for now, but not for the long term. I'm fine merging this in the short term, but in the long term I don't like the idea of having non-commands here. Postgres doesn't consider `WITH` its own command. Rather, it's a form of the ones we already know: select, insert, update, ....

So, please create an issue to rework the gen_psql_tests to slot the right "WITH" commands to the right standard command. I think we'll need to do something similar for `SELECT INTO` vs `SELECT`.

Once we have a correct grammar, reworking the tests should be easy. We just have to tweak the script, regenerate the test files and maybe tweak the Java harness a bit.